smooth/components/component_battery.py

```diff
@@ -37,26 +37,35 @@
 
 Maximum chargeable/dischargeable energy
 ---------------------------------------
-The maximum chargeable and dischargeable energy [Wh] per timestep is dependant on
-the C-rate and the state of charge of the battery. Because of the charging losses
-defined by charging efficiency, the nominal value of the input flow must be higher
-than what is actually entering the battery in order to ensure that the battery can
-be fully charged in one time step. Due to the *inflow_conversion_factor*
-parameter (in :func:`~smooth.components.component_battery.Battery.create_oemof_model`),
-the battery will then receive the right amount.
+The maximum chargeable or dischargeable power [W] going in to or out of the battery is dependant
+on the C-rate and the capacity:
 
 .. math::
-    E_{in,max} = \\frac{min(c_{r,charge} \\cdot C \\cdot \\frac{t}{60},
-    C - SOC \\cdot C)}{\\mu_{charge}} \n
-    E_{out,max} = min(c_{r,discharge} \\cdot C \\cdot \\frac{t}{60}, SOC \\cdot C)
+    P_{charge,max} = C * c_{r,charge}
 
-* :math:`E_{in,max}` = maximum chargeable energy [Wh]
-* :math:`c_{r,charge}` = C-Rate [-/h]
+    P_{discharge,max} = C * c_{r,discharge}
+To ensure that the battery can be charged and discharged within the time frame inherently defined
+by the C-rate (1C : Full capacity can be (dis-)charged within one hour), the nominal value of
+the input-flow/output-flow between bus and battery needs to include the energy losses during the
+(dis-)charging process:
+
+.. math::
+    P_{in,max} = P_{charge,max} / \\mu_{charge}
+
+    P_{out,max} = P_{discharge,max} * \\mu_{discharge}
+Due to the inflow_conversion_factor / outflow_conversion_factor
+(in :func:`~smooth.components.component_battery.Battery.create_oemof_model`)
+the battery will then receive right amount.
+
+* :math:`P_{charge,max}` = maximum chargeable power at the battery [W]
 * :math:`C` = storage capacity [Wh]
-* :math:`SOC` = state of charge [Wh]
+* :math:`c_{r,charge}` = C-Rate for charging [-/h]
+* :math:`P_{discharge,max}` = maximum dischargeable power at the battery [W]
+* :math:`c_{r,discharge}` = C-Rate for discharging [-/h]
+* :math:`P_{in,max}` = maximum nominal power flowing from bus to battery [W]
 * :math:`\\mu_{charge}` = charging efficiency [-]
-* :math:`E_{out,max}` = maximum dischargeable energy [Wh]
-* :math:`c_{r,discharge}` = C-Rate [-/h]
+* :math:`P_{out,max}` = maximum nominal power flowing from battery to bus [W]
```

Comment:
- not necessarily from/to **bus**

- maximum nominal power inflow to the battery [W]
- maximum nominal power outflow from the battery [W]

Reply:
[Battery component.pdf](https://github.com/rl-institut/smooth/files/5814217/Battery.component.pdf)
This is my understanding of the system, so the P_in is always from bus to battery e.g. How do you see it? Where else could the flow go if not from/to the bus?

Reply:
![Battery component](https://user-images.githubusercontent.com/39203376/104587829-d2038b80-5667-11eb-805b-65b6aa849e00.png)

Reply:
As well, I think it should be used this way. I just thought that theoretically it is possible to link a battery directly to another component and that it is easy to say it differently. However, looking at the single in-out-flow definition, this might not be useful, at least I do not see a good application for that.